### PR TITLE
Initial content from Alysha

### PR DESCRIPTION
**Pull request by Slimleashma**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -6,20 +6,423 @@
...
+========
+
+This document describes the procedure to pump and cool down LATISS from ambient pressure to its operational temperatures. 
+It also outlines how to perform a controlled warm-up in the case of servicing or programmed power cuts. 
+
+Pumping LATISS to vacuum
+========================
+
+Under atypical circumstances, LATISS needs to be warmed up to ambient temperature. 
+Large periods of power outage, severe earthquakes (`and different levels of recovery based on severity <https://tstn-027.lsst.io>`__), or technical intervention may have necessitated warm-up.
```
Comment (patrickingraham) [inline index.rst]: with different levels of recovery...?

File: index.rst [hunk 2]
```diff
@@ -6,20 +6,423 @@
...
+Pumping down, cooling, and powering up LATISS takes at least two days, and must be organized by ticket on the summit calendar and supported by daytime engineering support. 
+Mario Rivera, Craig Lage, and Alysha Shugart are points of contact for understanding and initiating the procedure. 
+
+Getting started
+---------------
+
+AuxTel must be under LOTO or the `E-STOP <https://obs-ops.lsst.io/Daytime-Operations/Auxiliary-Telescope/AuxTel-E-Stop-Procedure/E-Stop-Procecure.html>`__ must be engaged to perform this operation.
```
Comment (patrickingraham) [inline index.rst]: Are we still using the E-stop?

File: index.rst [hunk 3]
```diff
@@ -6,20 +6,423 @@
...
+Under atypical circumstances, LATISS needs to be warmed up to ambient temperature. 
+Large periods of power outage, severe earthquakes (`and different levels of recovery based on severity <https://tstn-027.lsst.io>`__), or technical intervention may have necessitated warm-up.
+Pumping down, cooling, and powering up LATISS takes at least two days, and must be organized by ticket on the summit calendar and supported by daytime engineering support. 
+Mario Rivera, Craig Lage, and Alysha Shugart are points of contact for understanding and initiating the procedure. 
+
+Getting started
+---------------
+
+AuxTel must be under LOTO or the `E-STOP <https://obs-ops.lsst.io/Daytime-Operations/Auxiliary-Telescope/AuxTel-E-Stop-Procedure/E-Stop-Procecure.html>`__ must be engaged to perform this operation. 
+The safety team is responsible for the the approval of the LOTO procedure, which will be linked on this page once it is affirmed.
```
Comment (patrickingraham) [inline index.rst:27]: Ok.. so this probably answers my question but I don't quite understand...

File: index.rst [hunk 4]
```diff
@@ -6,20 +6,423 @@
...
+ 
+Start by powering up the instrument cooling electronics, located in the first floor cabinet.
```
Comment (patrickingraham) [inline index.rst]: I'm worried that someone might include the cryotiger (poly cold) as "instrument cooling electronics" .. maybe rephrase to "powering up the instruments required to support the instrument cooldown"

File: index.rst [hunk 5]
```diff
@@ -6,20 +6,423 @@
...
+
+Getting started
+---------------
+
+AuxTel must be under LOTO or the `E-STOP <https://obs-ops.lsst.io/Daytime-Operations/Auxiliary-Telescope/AuxTel-E-Stop-Procedure/E-Stop-Procecure.html>`__ must be engaged to perform this operation. 
+The safety team is responsible for the the approval of the LOTO procedure, which will be linked on this page once it is affirmed.
+ 
+Start by powering up the instrument cooling electronics, located in the first floor cabinet. 
+It is important to make sure that the electronics are turned on but the outputs are disabled, so no control loops are operating on LATISS until it's under vacuum. 
+These electronics consist of:
```
Comment (patrickingraham) [inline index.rst]: The electronics to be powered on at this point in time consist of:

File: index.rst (hunk 4 above)
Comment (patrickingraham) [reply index.rst]: In fact, you might want to add some sort of `..important::` note that says the instrument can be damaged by prematurely powering on the polycold system and extreme caution must be exercised to ensure everything is ready before starting to cool.

File: index.rst [hunk 6]
```diff
@@ -6,20 +6,423 @@
...
+
+-  3 HMP2030 power supplies
+-	1 Keithley 6487 Voltage Source
+-	1 Advantech Uno-1483G-434AE rail controller (not used in the process)
+-	1 Cryocon temperature controller (for CCD sensor)
+
+.. figure:: /_static/power_supply_picometer.png
+   :name: power-supply-picometer
```
Comment (patrickingraham) [inline index.rst]: not sure it matters, but it's a picoammeter... meaning it measures pico-amps.

File: index.rst [hunk 7]
```diff
@@ -6,20 +6,423 @@
...
+AuxTel must be under LOTO or the `E-STOP <https://obs-ops.lsst.io/Daytime-Operations/Auxiliary-Telescope/AuxTel-E-Stop-Procedure/E-Stop-Procecure.html>`__ must be engaged to perform this operation. 
+The safety team is responsible for the the approval of the LOTO procedure, which will be linked on this page once it is affirmed.
+ 
+Start by powering up the instrument cooling electronics, located in the first floor cabinet. 
+It is important to make sure that the electronics are turned on but the outputs are disabled, so no control loops are operating on LATISS until it's under vacuum. 
+These electronics consist of:
+
+-  3 HMP2030 power supplies
+-	1 Keithley 6487 Voltage Source
+-	1 Advantech Uno-1483G-434AE rail controller (not used in the process)
```
Comment (patrickingraham) [inline index.rst]: Why do you say this isn't used?
This is the hcu for the ATCamera, no? It reports and publishes the sensor values etc.

File: index.rst (hunk 5 above)
Comment (patrickingraham) [reply index.rst]: Also, these get powered on by the PDU or a switch? I think you should probably list the PDU information just incase they're powered off there (which they probably should be).

File: index.rst [hunk 8]
```diff
@@ -6,20 +6,423 @@
...
+.. figure:: /_static/rail_and_cryocon_controller.png
+   :name: rail-and-cryocon-controller
+   :alt: Rail and cryocon controller
+
+   Advantech Rail controller (left) and Cryocon Temperature controller (right).
+
+Setting up the turbo (vacuum) pump
+----------------------------------
+
+The spectrograph needs to work in a good vacuum (~10−7 Torr). For reaching this vacuum, there are two devices involved:
```
Comment (patrickingraham) [inline index.rst]: the sensor (CCD), the spectrograph optics and all other parts are not under vacuum.

File: index.rst [hunk 9]
```diff
@@ -6,20 +6,423 @@
...
+
+.. figure:: /_static/turbo_pump_equipment.png
+   :name: turbo-pump-equipment
+   :alt: Turbo pump equipment
+
+   Pfeifer turbopump and accessories.
+
+The Pfeifer turbopump lives in the AuxTel storage container on Calibration Hill. 
+The turbo pump, vacuum hose, power supplies, and pressure sensor are stored together on the floor and first shelf of the storage container. 
+The cardboard storage box is labeled.
```
Comment (patrickingraham) [inline index.rst]: Sounds like a bit of a sentence fragment; maybe just add this to the previous sentence?
"The turbo pump, vacuum hose, power supplies, and pressure sensor are stored together on the floor and first shelf of the storage container in a labeled cardboard storage box"

File: index.rst [hunk 10]
```diff
@@ -6,20 +6,423 @@
...
+Mario Rivera also has a copy. 
+
+.. figure:: /_static/auxtel_storage.png
+   :name: auxtel-storage
+   :alt: Auxtel Storage
+
+   Turbo pump supplies located in the storage unit outside of the AuxTel dome.
+
+The ion pump (mounted on LATISS) doesn’t have enough capacity to evacuate the dewar from normal atmospheric pressure to working pressure, so the Pfeifer Turbopump has to be used first. 
+The ion pump should be started only after ~10−6 Torr is reached (as the high pressure could destroy the ion pump).
```
Comment (patrickingraham) [inline index.rst]: Consider:
...(as operating the ion pump at higher pressures significantly decreases it's operational lifespan).

File: index.rst [hunk 11]
```diff
@@ -6,20 +6,423 @@
...
+
+Connect the turbopump to the dewar
+-----------------------------------
+
+Put the turbopump in a stable position close enough to the dewar position (to be close to the flange connections). 
+An example setup is shown in Figure 5.
```
Comment (patrickingraham) [inline index.rst]: I think you want to use a reference here, something like,
as shown in :ref:`turbopump-setup`

Comment (patrickingraham) [reply index.rst]: bah, github jacks up the formatting, the name has to be surrounded by these things `

File: index.rst [hunk 12]
```diff
@@ -6,20 +6,423 @@
...
+   It is likely that the instrument will need to be rotated to +20 degrees to accommodate this. 
+   The telescope mount may also need to be rotated to accommodate all of the power supplies necessary to run the turbopump and vacuum sensors.
+
+.. figure:: /_static/turbopump_setup.png
+   :name: turbopump-setup
+   :alt: Turbopump setup
+
+   Left - Connecting the turbopump to the dewar. Right - Vacuum hose connection to the dewar. 
+
+Remove the vacuum covers, and clean the turbopump and the hose flange with isoporpyl alcohol (using gloves), to avoid introducing contaminants inside the turbopump and hose.
```
Comment (patrickingraham) [inline index.rst]: isopropyl

using gloves and new kimwipes?

File: index.rst [hunk 13]
```diff
@@ -6,20 +6,423 @@
...
+Connect the hose to the turbopump flange. 
+The hose has to be in a relaxed shape, with no turn or folds. 
+
+After checking the instrument dewar valve is closed, clean both the vacuum hose and instrument dewar flange with alcohol.
```
Comment (patrickingraham) [inline index.rst:108]: hmm, probably best to make sure the instrument valve is closed before starting to attach anything... but it better be closed or someone screwed up bad!

File: index.rst [hunk 14]
```diff
@@ -6,20 +6,423 @@
...
+Connect the other end of the hose to the dewar, without forcing the hose into a stressed position.
+This is why it may be necessary to rotate Nasmyth #2 between 0° and 20° to find a comfortable fit. 
+
+Check the turbopump output valve is closed.
```
Comment (patrickingraham) [inline index.rst:112]: turbopump output valve? I didn't think there was one?

Comment (patrickingraham) [reply index.rst:112]: There is a vent which has to be closed... which is a black thumbscrew where the pipe enters the red box... but that should never really be opened.

File: index.rst [hunk 15]
```diff
@@ -6,20 +6,423 @@
...
+
+.. figure:: /_static/instrument_dewar_valve.png
+   :name: instrument-dewar-valve
+   :alt: LATISS dewar valve
+
+   Picture of the LATISS dewar valve. 
+
+.. note:: 
+   The turbopump pump output valve is not typically used during the procedure. 
+   What is important is that the pressure in the vacuum line to be lower than the pressure of the dewar BEFORE opening the dewar valve.
```
Comment (patrickingraham) [inline index.rst]: ...BEFORE ever opening the dewar valve.

File: index.rst [hunk 16]
```diff
@@ -6,20 +6,423 @@
...
+
+.. figure:: /_static/pressure_sensor.png
+   :name: pressure-sensor
+   :alt: Turbopump pressure sensor
+
+   Left – Turbopump vacuum line sensor (connected to the top of the pump). Right – Pressure electronic display.
+
 .. note::
+   The setup in the left photo of Fig. 7 is example only. 
+   Please do not operate the Turbopump on a non-rigid surface.
```
Comment (patrickingraham) [inline index.rst]: Remove the "Please" -- just so it's clear that it's not a request...

File: index.rst [hunk 17]
```diff
@@ -6,20 +6,423 @@
...
+   :alt: Turbopump pressure sensor
+
+   Left – Turbopump vacuum line sensor (connected to the top of the pump). Right – Pressure electronic display.
+
 .. note::
+   The setup in the left photo of Fig. 7 is example only. 
+   Please do not operate the Turbopump on a non-rigid surface. 
 
-   **This technote is a work-in-progress.**
+Connect the turbopump to a secure electrical outlet, and turn the flip switch (at the back of the device on the lower right) to turn the pump on.
```
Comment (patrickingraham) [inline index.rst]: rocker switch

File: index.rst [hunk 18]
```diff
@@ -6,20 +6,423 @@
...
+   :name: operational-turbopump
+   :alt: Operational turbopump
+
+   Turbopump is fully spun-up.
+
+Before opening the instrument dewar valve, it’s necessary to have similar vacuums on the pump and inside the dewar. 
+If this is not the case, the turbopump or the dewar inside LATISS could receive a high pressure shock. 
+Ideally, you should wait until the pressure is lower on the turbopump's side.
+
+- If the dewar was evacuated completely, its pressure should be close to the turbopump at this point (~ atmospheric pressure).
```
Comment (patrickingraham) [inline index.rst]: I think this is ambiguous... "evacuated completely" means (to me) that it'll be at a really high vacuum, not at atmospheric pressure.
If you want the gory details, consider something like:
"If the dewar has been warmed up but not opened, the pressure will be approximately the partial pressure of water vapour (~20 Torr). In the case the dewar has been serviced or opened, it may be at atmospheric pressure or under a slight positive pressure if purged with dry nitrogen."
If not, then just replace "evacuated completely" with "warmed up to ambient temperature" or something like that.

**Review by patrickingraham — DISMISSED** [review]
Sorry, didn't quite finish this.
It's looking good. Just a bunch of minor stuff for the most part. I'll finish the review tomorrow.

File: index.rst (hunk 1 above)
Comment (Slimleashma) [reply index.rst]: Changed language. Clarified. 👍

File: index.rst (hunk 2 above)
Comment (Slimleashma) [reply index.rst]: Both, until I can publish the LOTO procedure for everyone to use. 👍

File: index.rst (hunk 3 above)
Comment (Slimleashma) [reply index.rst:27]: It needs to be under LOTO. Procedure forthcoming. I do not have approval to publish. 👍

File: index.rst (hunk 4 above)
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst (hunk 5 above)
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst (hunk 6 above)
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst (hunk 7 above)
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst (hunk 8 above)
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst (hunk 9 above)
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst (hunk 10 above)
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst (hunk 11 above)
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst (hunk 12 above)
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst (hunk 13 above)
Comment (Slimleashma) [reply index.rst:108]: 👍

File: index.rst (hunk 14 above)
Comment (Slimleashma) [reply index.rst:112]: That's what it is called. 👍

File: index.rst (hunk 15 above)
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst (hunk 14 above)
Comment (Slimleashma) [reply index.rst:112]: That's what it is called. 👍

File: index.rst (hunk 16 above)
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst (hunk 17 above)
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst (hunk 18 above)
Comment (Slimleashma) [reply index.rst]: 👍

**Comment by craiglagegit** [comment]
Looks good to me.  I am approving so it is ready to merge.

**Review by craiglagegit — APPROVED** [review]
